.github/workflows/release.yml

```diff
@@ -178,12 +178,68 @@ jobs:
       - name: package
         # Copy-Item -Verbose -Recurse -Path "models\*" -Destination "${{ env.PACKAGENAME }}"
         run: |
+          Import-Module "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\Microsoft.VisualStudio.DevShell.dll
+          Enter-VsDevShell -SkipAutomaticLocation -VsInstallPath "C:\Program Files\Microsoft Visual Studio\2022\Enterprise" -Arch amd64        
           mkdir ${{ env.PACKAGENAME }}
           Copy-Item -Verbose -Path "README.md" -Destination "${{ env.PACKAGENAME }}"
           Copy-Item -Verbose -Path "LICENSE" -Destination "${{ env.PACKAGENAME }}"
           Copy-Item -Verbose -Path "build\Release\${{ needs.setup.outputs.APPNAME }}.exe" -Destination "${{ env.PACKAGENAME }}"
-          Copy-Item -Verbose -Path "C:\windows\system32\vcomp140.dll" -Destination "${{ env.PACKAGENAME }}"
-          Copy-Item -Verbose -Path "C:\windows\system32\vcomp140d.dll" -Destination "${{ env.PACKAGENAME }}"
```

Comment:
We certainly can't do this for the ARM64 binary, so I fixed both x64 and arm64 to copy the OpenMP runtime using the  correct folder which should be the one that comes with the toolset and not the one running in the OS.

Also, out of curiosity, why copy the Debug binary since `upscayl-bin` is only built in Release configuration?